src/controllers/AdminController.ts

```diff
@@ -165,6 +165,41 @@ export class AdminAPI extends Controller {
     return template;
   }
 
+  /**
+   * Create a templates. An overview of Template usage in Retraced can be found at
+   *
+   * https://boxyhq.com/docs/retraced/advanced/display-templates
+   *
+   * @param auth          Base64 encoded JWT
+   * @param projectId     The project id
+   * @param environmentId The environment id
+   * @param body          The template resource to create
+   */
+    @Post("project/{projectId}/templates")
```

Comment:
@oliverqx We have a standard of using `/bulk`.
```suggestion
    @Post("project/{projectId}/templates/bulk")
```